indent/verilog_systemverilog.vim

```diff
@@ -65,16 +65,16 @@ if exists("*GetVerilogSystemVerilogIndent")
   finish
 endif
 
-if exists("g:verilog_disable_indent")
-  let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
-else
-  let s:verilog_disable_indent = []
-endif
-
 set cpo-=C
 
 function! GetVerilogSystemVerilogIndent()
 
+  if exists("g:verilog_disable_indent")
+    let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
+  else
+    let s:verilog_disable_indent = []
+  endif
+
```

Comment:
What do you have in mind?

Note that I'm trying to allow a user to dynamically change this configuration without the need to do anything else afterwards. Also, do you think this will make the code **that** slower, at least compared with the complexity of the indentation algorithm?

Also note that we're already doing something similar [here](https://github.com/vhda/verilog_systemverilog.vim/blob/indent/fix/indent/verilog_systemverilog.vim#L78-L93).